brands/amenity/fuel.json

```
@@ -3404,5 +3404,16 @@
       "brand": "福懋加油站",
       "name": "福懋加油站"
     }
+  },
+  "amenity/fuel|Giant Eagle": {
+    "matchNames": ["gaint eagle fuel"],
+    "nomatch": ["shop/supermarket|Giant Eagle"],
+    "tags": {
+      "amenity": "fuel",
+      "brand": "Giant Eagle",
+      "brand:wikidata": "Q153417",
```

Looks like you copied the wikidata value for Kroeger here...
```suggestion
      "brand:wikidata": "Q1522721",
```